functions/src/handlers/ApproveClassHandler.ts

```diff
@@ -17,10 +17,11 @@ export function approveclass (req, res) {
         admin.database().ref(`users/${uid}/universities/${universityName}`)
         .set(classList)
         .then(() => {
-        return res.status(200).send({message: 'Approved and added student to class.'})
+            return res.status(200).send({message: 'Approved and added student to class.'})
```

Comment:
You use single quote for message, but I and @hetobias use double quote for message, which one is the standard or does not master?

Reply:
Single quote is the standard. @hetobias  would you mind documenting this.